src/components/AccountIcon.js

```diff
@@ -49,17 +50,45 @@ export default function AccountIcon(props) {
 			.catch(console.error);
 	};
 
+	if (address === '') {
+		return (
+			<View>
+				<Image
+					source={network.logo}
+					style={{
+						backgroundColor: colors.bg_text_sec,
+						borderRadius: 40,
+						height: 40,
+						width: 40
+					}}
+				/>
+			</View>
+		);
+	}
+	if (address === 'new') {
```

Comment:
what is the reasoning for setting meta-info about an address in the address itself (L74 as well)? I guess this typechecks but it's quite confusing. A separate prop would be appropriate here IMHO.